docs/src/templates/pages/styleguide/ui-components/badges.hbs

```diff
@@ -0,0 +1,61 @@
+---
+title: Badges
+section-title: UI Components
+docs: true
+
+navgroup: styleguide
+navsub: ui-components
+navactive: badges
+---
+<p>Badges can be used to display small pieces of information.</p>
```

Comment:
Should probably have a sentence saying that it's an optional mixin in Fozzie that can be switched on.

Reply:
Done. I updated the `c-card` component description as well.